extra/debian/control

```diff
@@ -33,6 +33,7 @@ Build-Depends: curl,
                python-lzma,
                python-nose,
                python-opencv,
+               python-pysnmp4,
```

Comment:
You also need to add `python-pysnmp4` to the `Depends` section of the `stb-tester` package.

(Also leave this line in the `Build-Depends`, so that we can run `make check` when building the deb.)